### ISSUE TITLE
bug: with appraisal refining 'really strong' does not....

### ISSUE THREAD

**Issue by MartinParis** [farkam135/GoIV#684]
bug: with appraisal refining 'really strong' does not get recognized when the name of the pokemon drives the 'strong' string to the second line in the pogo appraisal pop up/message box.
![screenshot_20170512-113840](https://cloud.githubusercontent.com/assets/7744829/26005768/15be70ac-3708-11e7-931b-65fb87b94bec.png)
![screenshot_20170512-113904](https://cloud.githubusercontent.com/assets/7744829/26005812/38cb9f3e-3708-11e7-83d3-5d537f02c5ec.png)

**Comment by TripSixes** [comment]
Yes, this is an unfortunate side effect of the way that the OCR is pulling the text string and then how we are comparing one to the other.   I would venture that the proper solution here would be to see about manipulating the OCR'd string to properly "remove" any new-lines (can't remember how they are represented in the text string), so that the phrases we are looking for should match regardless of same line or wrapped-line.

For now, keep your Pokemon names short. ;)  _(kidding)_

**Comment by MartinParis** [comment]
Hi, minor issue already, but with latest update even more minor 😀, IV is
also downgraded in its effect.

Op 1 jul. 2017 07:37 schreef "Andrew Mc" <notifications@github.com>:
